test/test_maxwellDriver1d.py

```diff
@@ -37,15 +37,16 @@ def test_fdtd_pec():
     R = np.corrcoef(initialFieldE, -finalFieldE)
     assert R[0, 1] > 0.9999
 
-    # for _ in range(1000):
-    #     driver.step()
-    #     plt.plot(sp.x, driver['E'],'b')
-    #     plt.plot(sp.xH, driver['H'],'r')
-    #     plt.ylim(-1, 1)
-    #     plt.title(driver.timeIntegrator.time)
-    #     plt.grid(which='both')
-    #     plt.pause(0.01)
-    #     plt.cla()
+    driver['E'][:] = initialFieldE[:]
+    for _ in range(100):
+        driver.step()
+        plt.plot(sp.x, driver['E'],'b')
```

Comment:
Plots should not appear in tests... this is for debugging, you have left it uncommented.